specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/workbooks_API.json

```diff
@@ -0,0 +1,583 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Workbook"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The newly created workbook.",
+            "schema": {
+                "$ref": "#/definitions/Workbook"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkbookAdd": {
+            "$ref": "./examples/WorkbookAdd.json"
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates a workbook that has already been added.",
+        "operationId": "Workbooks_Update",
+        "parameters": [          
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkbookResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "workbookProperties",
+            "description": "Properties that need to be specified to create a new workbook.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Workbook"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The workbook definition updated.",
+            "schema": {
+                "$ref": "#/definitions/Workbook"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkbookList": {
+            "$ref": "./examples/WorkbookUpdate.json"
+          }
+        }
+      }
+    },
+    "/providers/microsoft.insights/workbooks/category/{category}": {  
+      "get": {
+        "description": "Gets a list of workbooks.",
+        "operationId": "Workbooks_ListBySourceId",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SourceIdParameter"
+          },
+          {
+            "$ref": "#/parameters/CategoryParameter"
+          },
+          {
+            "$ref": "#/parameters/TagsParameter"
+          },
+          {
+            "$ref": "#/parameters/CanFetchWorkbookContentParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A list containing 0 or more workbook definitions.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Workbook"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkbooksList": {
+            "$ref": "./examples/WorkbooksListLink.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroup/{resourceGroupName}/resourceName/{resourceName}/providers/microsoft.insights/workbooks": {  

```

Comment:
Is the intent of this API to create a workbook at any resource scope? The workbookName segment is missing in the URL.